gui/syncthing/core/eventService.js

```diff
@@ -18,14 +18,43 @@ angular.module('syncthing.core')
                 return;
             }
             $rootScope.$broadcast(self.ONLINE);
-
+            //GREEN
+            $("link[rel='shortcut icon']").attr("href","assets/img/favicon_idle.png")
```

Comment:
I suggest you do 

var statusIcon = $(...selector...);

and reuse that, as otherwise you are traversing the DOM every time you want to change something.